chart/values.yaml

```diff
@@ -0,0 +1,359 @@
+        name: back-storage
+  volumes:
+    - name: back-storage
+      persistentVolumeClaim:
+        claimName: back-storage
+  replicaCount: 1
+  podLabels: {}
+  deploymentAnnotations: {}
+  podAnnotations: {}
+  imagePullSecrets: []
+  service:
+    annotations: {}
+    labels: {}
+    type: ClusterIP
+  serviceAccount:
+    create: true
+    automount: true
+    annotations: {}
+    name: ~
+  extraContainers: []
+  extraInitContainers: []
+  extraVolumes: []
+
+front:
+  name: front
+  kyoo_front:
+    livenessProbe: {}
+    readinessProbe: {}
+    resources: {}
+    containerSecurityContext: {}
+    extraVolumeMounts: []
+    extraArgs: []
+    extraEnv: []
+    image:
+      repository: ~
+      tag: ~
+  replicaCount: 1
+  podLabels: {}
+  deploymentAnnotations: {}
+  podAnnotations: {}
+  imagePullSecrets: []
+  service:
+    annotations: {}
+    labels: {}
+    type: ClusterIP
+  serviceAccount:
+    create: true
+    automount: true
+    annotations: {}
+    name: ~
+  extraContainers: []
+  extraInitContainers: []
+  extraVolumes: []
+
+matcher:
+  name: matcher
+  kyoo_matcher:
+    livenessProbe: {}
+    readinessProbe: {}
+    resources: {}
+    containerSecurityContext: {}
+    extraVolumeMounts: []
+    # workaround until dedicated image is created
+    extraArgs: 
+      - matcher
+    extraEnv: []
+    image:
+      # workaround until dedicated image is created
+      repository: ghcr.io/zoriya/kyoo_scanner
+      tag: ~
+  replicaCount: 1
+  podLabels: {}
+  deploymentAnnotations: {}
+  podAnnotations: {}
+  imagePullSecrets: []
+  service:
+    annotations: {}
+    labels: {}
+    type: ClusterIP
+  serviceAccount:
+    create: true
+    automount: true
+    annotations: {}
+    name: ~
+  extraContainers: []
+  extraInitContainers: []
+  extraVolumes: []
+
+scanner:
+  name: scanner
+  kyoo_scanner:
+    livenessProbe: {}
+    readinessProbe: {}
+    resources: {}
+    containerSecurityContext: {}
+    extraVolumeMounts: []
+    extraArgs: []
+    extraEnv: []
+    image:
+      repository: ~
+      tag: ~
+  replicaCount: 1
+  podLabels: {}
+  deploymentAnnotations: {}
+  podAnnotations: {}
+  imagePullSecrets: []
+  service:
+    annotations: {}
+    labels: {}
+    type: ClusterIP
+  serviceAccount:
+    create: true
+    automount: true
+    annotations: {}
+    name: ~
+  extraContainers: []
+  extraInitContainers: []
+  extraVolumes: []
+
+transcoder:
+  name: transcoder
+  kyoo_transcoder:
+    livenessProbe: {}
+    readinessProbe: {}
+    resources: {}
+    containerSecurityContext: {}
+    extraVolumeMounts: []
+    extraArgs: []
+    extraEnv: []
+    image:
+      repository: ~
+      tag: ~
+    volumeMounts:
+      - mountPath: /metadata
+        name: transcoder-storage
+      - mountPath: /cache
+        name: cache
+  volumes:
+    - name: transcoder-storage
+      persistentVolumeClaim:
+        claimName: transcoder-storage
+    - name: cache
+      emptyDir: {}
+  replicaCount: 1
```

Comment:
Distributed transcoder still is not done yet, I see two ways of making this work:
- Share the transcoder's info cache (but I think this would come with a performance cost to sync instances)
- Making sure the same video file is always handled by the same transcoder + sending messages to others transcoder when a client starts a stream (so others transcoder can know the user left & cleanup the leftover transcode process).

For now (and as long as we don't do the first idea), the /cache should NOT be shared between instances.

Reply:
With this setup, the cache is stored between instances.  Need to do some more reading on best way to ensure each replica has its own cache... 

could just remove the mount and have it use the disk overlay.

Reply:
Resolving thread.  emptyDir creates a local cache that is not shared across pods.  This means if multiple replicas were running, each would have their own independent cache.  

Current cache writes to disk. Unsure what the performance requirements are for using a distributed cache.